great_expectations/validation_operators/validation_operators.py

```diff
@@ -235,6 +235,14 @@ def __init__(
                 )
             self.actions[action_config["name"]] = new_action
 
+    @property
+    def _cloud_mode(self) -> bool:
```

Comment:
Change name to `is_cloud_context` or something a bit different to better convey what is going on with this temp property?